src/main/java/net/fabricmc/loom/configuration/providers/minecraft/mapped/AbstractMappedMinecraftProvider.java

```diff
@@ -226,6 +226,7 @@ private void remapJar(RemappedJars remappedJars, ConfigContext configContext) th
 		} catch (Exception e) {
 			throw new RuntimeException("Failed to remap JAR " + remappedJars.inputJar() + " with mappings from " + mappingConfiguration.tinyMappings, e);
 		} finally {
+			project.getLogger().lifecycle(":remapping minecraft (TinyRemapper, " + fromM + " -> " + toM + ")");
```

Comment:
```suggestion
			project.getLogger().info(":remapping minecraft (TinyRemapper, " + fromM + " -> " + toM + ")");
```
I don't think we need this log clutter back, if anything we should drop the log levels to info in some of the other cases of this

Reply:
I was doing this for debugging earlier, I'll drop it down to info